recipes/julia/meta.yaml

```diff
@@ -0,0 +1,57 @@
+package:
+  name: julia
+  version: "0.5.2"
+
+source:
+  fn: v0.5.2.tar.gz
+  url: https://github.com/JuliaLang/julia/archive/v0.5.2.tar.gz
+  sha256: efa3303d63e39bf3ea06e23ef68807cca0095c7e0507bdf26eae1d5dee7868c7
+  patches:
+  - rpath.patch
+
+build:
+  skip: True  # [osx]
+  skip: True  # [py3k]
+  rpaths:
+    - lib/julia/
+  number: 0
+
+requirements:
+  build:
+    - gcc  # [linux]
+    - llvm  # [osx]
+    - cmake
+    - zlib
+    - python
+    - autoconf 2.69 pl5.*
+    - automake 1.15 pl5.*
+    - openblas
+    - llvmdev 3.9.*
+    - fftw
+    - gmp 0.5.*
+    - mpfr
+    - curl
+    - libssh2 1.7.*
+    - libxml2
```

Comment:
please pin this to `libxml2 2.9.*`
A complete list can be found here: https://github.com/conda-forge/conda-forge.github.io/blob/0661e2ecf4b213ddf36350ca100a5a820eb17eb7/scripts/pin_the_slow_way.py#L40-L85